addons/calendar/models/calendar_event.py

```diff
@@ -685,6 +730,12 @@ def action_open_composer(self):
             'context': compose_ctx,
         }
 
+    def action_join_video_call(self):
+        return {
+            'type': 'ir.actions.act_url',
+            'url': self.videocall_location,
+        }
```

Comment:
Functional question: for internal users, we might want to open the channel on discuss (or maybe even as chat window) instead of opening the external page?

Reply:
You're right. We should change this method to differentiate the behavior from discuss and other websites.